tests/functional/lib/helpers.js

```diff
@@ -246,6 +246,19 @@ define([
           .execute(function (verificationLink, windowName) {
             var newWindow = window.open(verificationLink, windowName);
 
+            // Hook up the new window to listen for WebChannel messages.
+            // XXX TODO: this is pretty gross to do universally like this...
+            // XXX TODO: it will go away if we can make the original tab
+            //           reliably be the one to complete the oauth flow.
```

Comment:
> Why'd he have to tie that to jQuery?

Does not have to be, I saw the brutal example on MDN and looked for a contained solution.